Source/controls/touch/gamepad.cpp

```diff
@@ -1,5 +1,7 @@
 #include <SDL.h>
 
+#ifndef USE_SDL1

```

Comment:
Why not just put this at the very top of the file?

Reply:
Removed this and went with only conditionally including things instead